ietf/doc/utils.py

```diff
@@ -936,6 +936,8 @@ def get_predecessors(doc, predecessors=None):
                 if document not in predecessors:
                     predecessors.append(document)
                     predecessors.extend(get_predecessors(document, predecessors))
+        if doc.came_from_draft():
```

Comment:
See comment in `views_doc` - I think this and the `became_rfc()` call below are making extra DB hits by testing and appending separately